cfpb/sbl-frontend
# feat(skip-links): add skip link and main id to missing pages

Currently we don't have skip links that aid in making our content accessible to those using tab navigation.

Closes: #359 

## Changes

- adds skip link to App.tsx
- adds missing `id="main"` to pages that still need it

## How to test this PR

1. Can you press `tab` then `enter` to skip to the main content of a page in the tab order? (Try pressing `tab` again to confirm.

## Screenshots
<img width="533" alt="Screenshot 2024-04-02 at 3 35 55 AM" src="https://github.com/cfpb/sbl-frontend/assets/19983248/fb528a31-ad7e-4dc7-b941-dd9f8f786781">


## Notes

- the skip link could be a little fancier with its CSS transition like on cf.gov, but we're prioritizing [just the accessibility first](https://github.com/cfpb/design-system-react/pull/330)
- works in conjunction with https://github.com/cfpb/design-system-react/pull/330 for the focus ring
